autotest/test_atm.py

```diff
@@ -45,13 +45,13 @@ def test_init(self, domain):
 
         # check the values
         atol = 1e-4
-        assert np.isclose(sun_hrs_ans, solar_geom.sun_hrs, atol=atol).all()
+        assert np.isclose(solar_geom.sun_hrs, sun_hrs_ans, atol=atol).all()
```

Comment:
The order of np.allclose/isclose is not symmetric with the "reference" belonging to the second argument as in diff = val - ref